<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>poky - Poky Build Tool and Metadata</title>
<meta name='generator' content='cgit v0.10.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://git.yoctoproject.org/cgit/cgit.cgi/poky/atom/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/cgit.cgi/'><img src='/cgit/yocto-project-transp.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/cgit.cgi/'>index</a> : <a title='poky' href='/cgit/cgit.cgi/poky/'>poky</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='1.1_M1'>1.1_M1</option>
<option value='1.1_M2'>1.1_M2</option>
<option value='1.1_M3'>1.1_M3</option>
<option value='1.1_M4'>1.1_M4</option>
<option value='1.2_M1'>1.2_M1</option>
<option value='1.2_M2'>1.2_M2</option>
<option value='1.2_M3'>1.2_M3</option>
<option value='1.2_M4'>1.2_M4</option>
<option value='1.3_M1'>1.3_M1</option>
<option value='1.3_M2'>1.3_M2</option>
<option value='1.3_M3'>1.3_M3</option>
<option value='1.3_M4'>1.3_M4</option>
<option value='1.3_M5'>1.3_M5</option>
<option value='1.3_beta'>1.3_beta</option>
<option value='1.4_M1'>1.4_M1</option>
<option value='1.4_M3'>1.4_M3</option>
<option value='1.4_M4'>1.4_M4</option>
<option value='1.4_M5'>1.4_M5</option>
<option value='1.6_M5'>1.6_M5</option>
<option value='bernard'>bernard</option>
<option value='blinky'>blinky</option>
<option value='clyde'>clyde</option>
<option value='daisy'>daisy</option>
<option value='daisy-next'>daisy-next</option>
<option value='danny'>danny</option>
<option value='danny-next'>danny-next</option>
<option value='denzil'>denzil</option>
<option value='dora'>dora</option>
<option value='dora-toaster'>dora-toaster</option>
<option value='dylan'>dylan</option>
<option value='edison'>edison</option>
<option value='elroy'>elroy</option>
<option value='green'>green</option>
<option value='laverne'>laverne</option>
<option value='master' selected='selected'>master</option>
<option value='master-next'>master-next</option>
<option value='pinky'>pinky</option>
<option value='purple'>purple</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>Poky Build Tool and Metadata</td><td class='sub right'>git repository hosting</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/cgit.cgi/poky/about/'>about</a><a href='/cgit/cgit.cgi/poky/'>summary</a><a href='/cgit/cgit.cgi/poky/refs/'>refs</a><a href='/cgit/cgit.cgi/poky/log/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>log</a><a class='active' href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>tree</a><a href='/cgit/cgit.cgi/poky/commit/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>commit</a><a href='/cgit/cgit.cgi/poky/diff/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>diff</a><a href='/cgit/cgit.cgi/poky/stats/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/cgit.cgi/poky/log/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/cgit.cgi/poky/tree/'>root</a>/<a href='/cgit/cgit.cgi/poky/tree/meta'>meta</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core'>recipes-core</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus'>dbus</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2'>dbus-glib-0.100.2</a>/<a href='/cgit/cgit.cgi/poky/tree/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>obsolete_automake_macros.patch</a></div><div class='content'>blob: 40e3b12b5dfeb3ef4daa792f6579dcc7764539c5 (<a href='/cgit/cgit.cgi/poky/plain/meta/recipes-core/dbus/dbus-glib-0.100.2/obsolete_automake_macros.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
</pre></td>
<td class='lines'><pre><code>Upstream-Status: Submitted [https://bugs.freedesktop.org/show_bug.cgi?id=59003]

Signed-off-by: Marko Lindqvist &lt;cazfi74@gmail.com&gt;
diff -Nurd dbus-glib-0.100/configure.ac dbus-glib-0.100/configure.ac
--- dbus-glib-0.100/configure.ac	2012-06-25 19:26:39.000000000 +0300
+++ dbus-glib-0.100/configure.ac	2013-01-03 04:53:22.314976758 +0200
@@ -8,7 +8,7 @@

 AM_INIT_AUTOMAKE([1.9])

-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)

 # Honor aclocal flags
 ACLOCAL="$ACLOCAL $ACLOCAL_FLAGS"
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.1 at 2014-07-21 11:20:23 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
